pkg/image/validation/version.go

```diff
@@ -21,22 +21,61 @@ func validateVersion(ctx *image.Context) []FailedValidation {
 		}
 	}
 
-	if definition.APIVersion == "1.0" && apiVersionsDefined {
-		failures = append(failures, FailedValidation{
-			UserMessage: "Helm chart APIVersions field is not supported in EIB version 1.0, must use EIB version 1.1",
-		})
-	}
+	if definition.APIVersion == "1.0" {
```

Comment:
Can we explore some generic way of doing this? It is clearly not going to scale well with 1.3 and higher.

Perhaps a switch statement for the version and then a map of keys and fields which are only supported from a specific version forward.

Reply:
Submitted the changes below.